pages/science/reactors/extensiblereactor.rst

```diff
@@ -0,0 +1,160 @@
+.. title: Extensible Reactors in Cantera
+.. has_math: true
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Extensible Reactor</h1>
+
+   .. class:: lead
+
+      This page defines and links to an example of an Extensible Reactor.
+
+Extensible Reactors
+*******************
+
+In some cases, Cantera's existing governing equations are insufficient 
+in describing a certain configuration. In this situation, Cantera 
+Reactors' governing equations can be modified to the user's specific 
+needs. An Extensible Reactor allows modifications of a Reactor class' 
+`governing equations </science/reactors/reactors.html>`__. If the governing equations 
+must be entirely replaced or an integrator besides CVODES is desired, a 
+`Custom Reactor </science/reactors/customreactor.html>`__
+may be a more appropriate Reactor class to use.
+
+The state variables available for modification 
+depend on the type of Reactor base class chosen. For example, choosing 
+an `Ideal Gas Constant Pressure Reactor </science/reactors/
+idealgasconstpresreactor.html>`__ allows
```

Comment:
I'd suggest using a `:doc:` or `:ref:` style reference here, to refer to the document or section you want to link to, instead of the absolute URL.

Reply:
I'm not sure how to use `:ref:` to reference across `.rst` files... I poked around and the examples/documentation was for referencing headers within the file.

Reply:
No problem, we can fix it later.